spring-boot/admin/src/main/resources/application.yml

```diff
@@ -23,6 +23,7 @@ ygg:
     verification.server.url: http://localhost:8080
 
   admin:
+    payable.enable: false
```

Comment:
There is already an existing key `payment.commission`, which is directly related, so I would suggest to rename this as `payment.enable` (so that it uses same prefix), and move it next to the other setting.